### DISCUSSION TITLE
Is there a way to get/print the configs loaded for an IKE connection in strongswan?

### DISCUSSION THREAD

**Discussion by Kannan-1989** [strongswan/strongswan#2081] [Q&A]
If the connection is not established between peers, then is there a way to print the configs loaded in my system for both IKE and child SA? This information is useful for debugging if the negotiation fails. Also, it is useful to cross-verify if the configs that I have provided and what is actually loaded in strongswan are same or not.

**Comment by Gor1234567** [comment]
You may set the debug level of 'cfg' subsystem up to 4 in /etc/strongswan.d/charon-logging.conf
